public/js/listeners.js

```diff
@@ -144,7 +144,7 @@ function startListeners() {
             }
         }
 
-        if (e.keyCode == 8 || e.key == "Delete") {
+        if (e.keycode == 8 || e.keycode == 46) {
```

Comment:
Why did u make `keycode` ? I don't think this exists. Use https://keycode.info/ to debug.

Reply:
@satu0king I made keycode == 46 so as to enable delete key

Reply:
Read the comment in the main issue.